db/seeds.rb

```diff
@@ -131,6 +131,23 @@
 
 ban_number_of_strikes_setting.save
 
+# Link to auction regulations on strikes
+violations_count_regulations_description = <<~TEXT.squish
+  Link to ToC clause on user agreement termination, used in ban message banner.
+  Must be single URL or hash of <locale>:<URL> elements.
+  URL can be relative ('/public/terms_and_conditions.pdf') 
+  or absolute ('https://example.com/terms_and_conditions.pdf'). Relative URL must start with a
+  forward slash.
+  Default: https://example.com#some_anchor
+TEXT
+
+violations_count_regulations_setting = Setting.new(code: :violations_count_regulations_link,
+                                            value: "https://example.com#some_anchor",
+                                            description: violations_count_regulations_description)
+
+violations_count_regulations_setting.save
```

Comment:
`save!` might save debugging time if something goes wrong and a record will not be saved.